frontend/src/lib/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -125,8 +125,9 @@ export function CodeSnippet({
                 <LemonButton
                     data-attr="copy-code-button"
                     icon={<IconCopy />}
-                    onClick={() => {
+                    onClick={(e) => {
                         if (text) {
+                            e.stopPropagation()
```

Comment:
otherwise clicking to copy text also triggers parent behaviour... in the case of player inspector collapsing the row